src/sagemaker_pytorch_serving_container/default_inference_handler.py

```diff
@@ -62,12 +82,22 @@ def default_predict_fn(self, data, model):
 
         Returns: a prediction
         """
-        device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
-        model.to(device)
-        input_data = data.to(device)
-        model.eval()
         with torch.no_grad():
-            output = model(input_data)
+            if os.getenv(INFERENCE_ACCELERATOR_PRESENT_ENV) == 'true':
+                logger.info(
```

Comment:
I noticed that in MXNet, there isn't any logged message indicating that EI is used. But, I think it is useful to have a logging statement like this since then the customer can definitively know that they're using EI. Wanted to point out the discrepancy, but I am onboard with this change
https://github.com/aws/sagemaker-mxnet-serving-container/blob/master/src/sagemaker_mxnet_serving_container/default_inference_handler.py#L62

Reply:
Logging per-inference is a lot of logging. May be ok for DEBUG level but I'd suggest removing the log.

Reply:
I agree that logging with each inference request is unnecessary, so I removed them.